scripts/check-newsfragment

```diff
@@ -16,7 +16,7 @@ pr="$PULL_REQUEST_NUMBER"
 
 # Print a link to the contributing guide if the user makes a mistake
 CONTRIBUTING_GUIDE_TEXT="!! Please see the contributing guide for help writing your changelog entry:
-https://github.com/matrix-org/matrix-appservice-irc/blob/develop/CONTRIBUTING.md#making-pull-requests"
+https://github.com/matrix-org/matrix-appservice-bridge/blob/develop/CONTRIBUTING.md#making-pull-requests"
```

Comment:
The `#making-pull-requests` anchor doesn't exist on that page.  What does exist is `#%EF%B8%8F-pull-requests` (which I got by right-clicking next to it)